.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -1,46 +1,46 @@
 name: "Bug Report"
 description: "Create a report for an issue you've encountered in the app."
-labels: ["bug"]
+labels: [ "bug" ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)